### PR TITLE
Add typings for node-gamepad

### PR DESCRIPTION
**Pull request by Lange**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

## Problems to resolve before merging
The published name of this package on npm is [`gamepad`](https://www.npmjs.com/package/gamepad), not `node-gamepad`. However, there [already exists a set of typings under that name, for the native browser gamepad API](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/gamepad). I am unsure how to resolve this conflict.

### CODE REVIEW

File: types/node-gamepad/tslint.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "noImplicitAny": true,
```
Comment (sheetalkamat) [inline types/node-gamepad/tslint.json]: tslint should contain only extends clause. Nothing else.

Comment (Lange) [reply types/node-gamepad/tslint.json]: Got it, the checklist item from the PR template made me think I might have to also manually add these items.

**Comment by Lange** [comment]
Bit of a shame that @typescript-bot doesn't acknowledge the commit I've already pushed.

**Comment by DanielRosenwasser** [comment]
Thanks for the PR!

**Comment by Lange** [comment]
@DanielRosenwasser Thanks for merging, but is the concern I brought up at the bottom of the original post something we need to worry about?

Copy/pasted again for ease of reference:
>The published name of this package on npm is gamepad, not node-gamepad. However, there already exists a set of typings under that name, for the native browser gamepad API. I am unsure how to resolve this conflict.

**Comment by DanielRosenwasser** [comment]
Whoops, sorry; okay, can you send a follow-up PR that

1. Renames `gamepad` to `w3-gamepad`, and
2. Renames `node-gamepad` to `gamepad`?

**Comment by DanielRosenwasser** [comment]
I think the impact will be fairly limited; `@types/gamepad` gets ~12 downloads a week.

**Comment by Lange** [comment]
Sorry for the delay @DanielRosenwasser, going to work on this now.

**Pull request merged**